packages/odyssey-react/src/components/Box/_box-root.scss

```diff
@@ -21,4 +21,158 @@
   line-height: $base-line-height;
   font-feature-settings: "lnum", "pnum";
   text-decoration-skip-ink: all;
+
+  // Catch-all border-box bad boy
+  /* stylelint-disable selector-max-universal */
+  * {
+    box-sizing: border-box;
+
+    &::before,
+    &::after {
+      box-sizing: inherit;
+    }
+  }
+  /* stylelint-enable selector-max-universal */
+
+  // Huge selector list layout reset
+  /* stylelint-disable selector-max-type */
+  div,
+  span,
+  applet,
+  object,
+  iframe,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  p,
+  blockquote,
+  pre,
+  a,
+  abbr,
+  acronym,
+  address,
+  big,
+  cite,
+  code,
+  del,
+  dfn,
+  em,
+  img,
+  ins,
+  kbd,
+  q,
+  s,
+  samp,
+  small,
+  strike,
+  strong,
+  sub,
+  sup,
+  tt,
+  var,
+  b,
+  u,
+  i,
+  center,
+  dl,
+  dt,
+  dd,
+  ol,
+  ul,
+  li,
+  fieldset,
+  form,
+  label,
+  legend,
+  table,
+  caption,
+  tbody,
+  tfoot,
+  thead,
+  tr,
+  th,
+  td,
+  article,
+  aside,
+  canvas,
+  details,
+  embed,
+  figure,
+  figcaption,
+  footer,
+  header,
+  hgroup,
+  menu,
+  nav,
+  output,
+  ruby,
+  section,
+  summary,
+  time,
+  mark,
+  audio,
+  video {
+    margin-block: 0;
+    margin-inline: 0;
+    padding-block: 0;
+    padding-inline: 0;
+    border: 0;
+    font: inherit;
+    font-size: 100%;
+    vertical-align: baseline;
+  }
+
+  button,
+  input {
+    font: inherit;
+    font-size: 100%;
+  }
```

Comment:
these types of styles should be added to <Button> and form input components as necessary rather than live here in Box. Lots of other similar reset styles like that we need to cleanup out of this commit too

Reply:
If someone creates a new component (or when we refactor ours) to use Box, shouldn't these element-specific normalizations be applied by default?

That is, they're specific to standardizing the *element* across browsers, not a specific component.

Reply:
@edburyenegren-okta that is a separate concern. What I'm talking about here is where specific styles should live. What was previously one single global reset stylesheet should be broken up into smaller parts where the styles live closer to the components they are targeting where applicable (like in this case with button and input) and otherwise they'd be captured by lower level utilities like Box and Text.

I don't agree we should be writing styles for raw button or input elements and including them as part of Box that is not the intention of this component

Reply:
This isn't a separate concern, as I'm talking about first-class use of Box, not styling a child. 

Ex: having a `Button` component doesn't mean that `button` will not be used as the root of a different component. If a user is writing a bespoke component or using Box for any other purpose, they might utilize `<Box as="button">`.

If we know there are cross-browser normalizations that need to take place in all instances of an element (regardless of the element), they should apply to that use case of Box unless specifically overridden.

Reply:
If there's a concern about selector list length, we could probably roll these resets into the default Box styling to avoid needing separate selectors here (they'll have `.root` either way). We'd want to test that to ensure we're not creating universal inheritance that isn't wanted.

One maintenance benefit to keeping them element-specific is that we can remove them per-element as normalizations become less necessary. 

Ex: Our IE 11 styles for HTML5 elements might be safe to remove at this point. It'd be harder to untangle the purpose of that code if it were all on `.root`.